app/controllers/users_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
     else
       @users = User.joins(:role).where.not(id: User.first.id) #Remove first user
     end
+    @users = User.search(@users, params[:search]) if params[:search]
```

Comment:
Doing that would likely just increase the number of lines of code. I would need to still keep the `flash[:alert]`s in the controller, and therefore would need to make conditionals for them both in the controller and in the model. It would be much better to just keep roles and badges within the controller.